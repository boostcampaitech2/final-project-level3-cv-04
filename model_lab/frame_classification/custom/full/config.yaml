root : "../../input"
train_csv : "full_train.csv"
valid_csv : "full_valid.csv"
batch : 256
lr : 0.01
epoch : 50
num_classes : 6

seed : 42

output_path : "../../output/frame_classification"
output_dir : "full"
save_capacity : 5

wandb : True
wandb_project : "handwash"
wandb_entity : "cv4"
wandb_group : "full"
log_freq : 10

class_names : ["Step 1","Step 2","Step 3","Step 4","Step 5","Step 6"]