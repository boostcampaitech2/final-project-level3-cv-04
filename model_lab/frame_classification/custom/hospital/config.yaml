root : "../../input"
train_csv : "train.csv"
valid_csv : "valid.csv"
batch : 256
lr : 0.01
epoch : 30
num_classes : 6

seed : 42

output_path : "../../output/frame_classification"
output_dir : "hospital"
save_capacity : 5

wandb : True
wandb_project : "handwash"
wandb_entity : "cv4"
wandb_group : "hospital"
log_freq : 10

class_names : ["Step 1","Step 2","Step 3","Step 4","Step 5","Step 6"]