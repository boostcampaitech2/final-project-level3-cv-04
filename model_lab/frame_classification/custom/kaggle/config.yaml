root : "../../input"
train_csv : "kaggle_train.csv"
valid_csv : "kaggle_valid.csv"
batch : 256
lr : 0.01
epoch : 50
num_classes : 6

seed : 42

output_path : "../../output/frame_classification"
output_dir : "kaggle"
save_capacity : 5

wandb : True
wandb_project : "handwash"
wandb_entity : "cv4"
wandb_group : "kaggle"
log_freq : 10

class_names : ["Step 1","Step 2","Step 3","Step 4","Step 5","Step 6"]