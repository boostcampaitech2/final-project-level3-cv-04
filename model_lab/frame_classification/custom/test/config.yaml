root : "../../input"
train_csv : "train.csv"
valid_csv : "valid.csv"
batch : 256
lr : 0.001 
epoch : 20
num_classes : 7