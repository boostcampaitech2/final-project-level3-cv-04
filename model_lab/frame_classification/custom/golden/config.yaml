root : "../../input"
# train_csv : "1216train.csv"
# valid_csv : "1216valid.csv"
train_csv : "good_full.csv"
valid_csv : "golden_label.csv"
batch : 32
lr : 0.01
epoch : 50
num_classes : 6

seed : 42

output_path : "../../output/frame_classification"
output_dir : "lastvalid_1216"
save_capacity : 5

wandb : True
wandb_project : "handwash"
wandb_entity : "cv4"
wandb_group : "golden"
log_freq : 10

class_names : ["Step 1","Step 2","Step 3","Step 4","Step 5","Step 6"]
